### ISSUE TITLE
platform.dist() removed in Python 3.8

### ISSUE THREAD

**Issue by Pixadus** [pop-os/system76-driver#134]
**Issue/Bug Description:**

System76-driver will not launch due to `platform.dist()` and `platform.linux_distribution()` functions having been removed from the `platform` module with the release of Python 3.8.

**Steps to reproduce (if you know):**

Using Python version 3.8:

```
$ python --version
Python 3.8.0
```

Running `sudo system76-driver` results in:

```
Traceback (most recent call last):
  File "/usr/bin/system76-driver", line 62, in <module>
    log.info('OS: %r', platform.dist())
AttributeError: module 'platform' has no attribute 'dist'
```

**Expected behavior:**

System76-driver should simply run normally. 

**Other Notes:**

[The Python docs on platform](https://docs.python.org/2/library/platform.html#unix-platforms ) state that this command has been replaced with those from the `distro` package. (See [the API docs on distro](https://distro.readthedocs.io/en/latest/)). 

Since `platform.linux_distribution()` is already being deprecated with the release of Python 3.5, we might want to update this.

**Comment by daveseff** [comment]
+1

**Comment by crawfxrd** [comment]
Resolved by: #144
